### ISSUE TITLE
Links in the website fail at accessibility contrast

### ISSUE THREAD

**Issue by stof** [async-aws/aws#408]
The green color used for links currently (`#429d25`) has a contrast ratio of 3.31 with white. This fails the accessibility criteria which requires at least a 4.5 ratio for text.
